Update Duckquill up to 4.4.0

BANOnotIT opened the pull request:
Closes #1 

@daudix I wasn't able to get 4.5.0 working because of https://codeberg.org/daudix/duckquill/issues/55

daudix committed:
Update Duckquill from v1.0.0 to v4.2.1

daudix committed:
Keep the dates

BANOnotIT committed:
build(deps): Bump duckquill up to 4.4.0

BANOnotIT committed:
fix: Use non transparent menu

BANOnotIT committed:
build: Enable reading time calculation

daudix commented:
The nav background styles don't work because it only accepts --nav-bg to be a single color. Instead you can override the `background` shorthand on `#site-nav`.

BANOnotIT commented:
Yeah, I know, and I use `background: var(--nav-bg)` [already](https://github.com/BANOnotIT/banonotit.github.io/blob/0ecd6dd9ee5363d900ca2ee74e126764d79a72ba/sass/custom.scss#L28). The problem is that I wanted to tweak other colors to my liking. So I just implemented the mixin myself and updated to 4.5.0.
